wp-content/themes/k1-theme/templates/page-getstarted.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Page: Get Started
+ */
+
+k1_enqueue_page_assets( 'getstarted' );
+?>
+<main>
+		<section class="first-section">
+			<div class="container-fluid">
+				<div class="row d-flex justify-content-center">
+					<div class="col-8">
+						<h2 class="headline"><span class="text-white">what are your ministry's greatest needs?</span></h1>
+						<h4>Tell Us About Your Organization, And We'll Connect You With  Ministry Support As Soon As Possible.</h4>
+                    </div>
+				</div>
+			</div>
+		</section>
+
+        <aside class="quotes row">
+		<?php $content->get_color_background_layers( 'quotes', 'zig-zag-right', array( 'k1-retreat-01', 'webp' ) ); ?>
+            <div class="swiper text-center" id="quotes-swiper">
```

Comment:
lines 22-55 can be replaced with the single function `<?php get_template_part('template-parts/sliders/swiper','quotes');?>`